tests/cases/classes/method-declaration

```diff
@@ -0,0 +1,30 @@
+---DESCRIPTION---
+Methods must not have unnecessary whitespace
+---FILENAME---
+Foo.php
+---CONTENTS---
+<?php
+
+namespace Vendor;
+
+class Foo
+{
+    public function bar  ( $baz  ,&$qux,  bool$quux, ? string  $quuz    =null   ,  string...$corge   ):
+    void
+    {
+    }
```

Comment:
Can't include an `abstract` example yet, waiting for https://github.com/squizlabs/PHP_CodeSniffer/pull/2148 to be merged/released.